gulpfile.js

```diff
@@ -83,75 +89,35 @@ function swatchJsonTransformer( json ) {
             for ( const [ name, meta ] of Object.entries(group.colorSystem) ) {
                 colorSystem.push({ name: name, value: meta.value });
             }
-        } else {
+        } else  {
             if (group.variables) {
                 for ( const [ name, meta ] of Object.entries(group.variables) ) {
                     variables.push({ name: name, value: meta.value });
                 }
             }
-
-            if (group.universal) {
-                for ( const [ name, meta ] of Object.entries(group.universal) ) {
-                    universal.push({ name: name, value: meta.value });
-                }
-            }
         }
     });
 
-    const templates = {
-        modern: () => {
-            const sassContent = [];
-
-            sassContent.push(`@forward "../scss/index.scss" with (`);
-            sassContent.push(colorSystem.map( (variable) => `\t$${variable.name}: ${variable.value} !default,`).join( '\n' ));
-
-            if ( colorsMap.length ) {
-                sassContent.push(`\t$kendo-colors: (`);
-                sassContent.push(colorsMap.map( (color) => `\t${color.name}: ${color.value},`).join( '\n' ));
-                sassContent.push(`\t) !default,`);
-            }
-
-            // Universal variables are also included here as they are part of the a11y swatch
-            sassContent.push(universal.map( (variable) => `\t$${variable.name}: ${variable.value} !default,`).join( '\n' ));
+    sassContent.push(`@forward "../scss/index.scss" with (`);
+    sassContent.push(colorSystem.map( (variable) => `\t$${variable.name}: ${variable.value} !default,`).join( '\n' ));
 
-            sassContent.push(`);\n`);
-
-            sassContent.push(`@use "../scss/index.scss" as *;\n`);
-
-            sassContent.push(`@include kendo-theme--styles();`);
-
-            return sassContent.join( '\n' );
-        },
-        // TODO remove legacy
-        legacy: () => {
-            const sassContent = [];
-            if ( !colorSystem.length ) {
-                sassContent.push(`$kendo-enable-color-system: false;\n`);
-            }
-            sassContent.push(colorSystem.map( (variable) => `$${variable.name}: ${variable.value};`).join( '\n' ));
+    if ( colorsMap.length ) {
+        sassContent.push(`\t$kendo-colors: (`);
+        sassContent.push(colorsMap.map( (color) => `\t${color.name}: ${color.value},`).join( '\n' ));
+        sassContent.push(`\t) !default,`);
+    }
 
-            if ( colorsMap.length ) {
-                sassContent.push(`\n$kendo-colors: (`);
-                sassContent.push(colorsMap.map( (color) => `\t${color.name}: ${color.value},`).join( '\n' ));
-                sassContent.push(`);\n`);
-            }
+    // Universal variables are also included here as they are part of the a11y swatch
```

Comment:
I'm not sure if we need the comment about the universal variables here as it's kind of misleading